extensions/calDotCom/actions/__tests__/getBooking.ts

```diff
@@ -1,41 +1,188 @@
-import { getBooking } from '../'
-jest.mock('../../calComApi')
+import { getBooking } from '../getBooking'
+import { faker } from '@faker-js/faker'
```

Comment:
adding `faker` just to generate uuids feels a bit unnecessary
`crypto.randomUUID()` should do the job

Reply:
I like having faker in as dev dependency, we should be example how to write specs. Plus, it's pretty much a standard for fake data generation in tests.